cmd/stack_create_test.go

```diff
@@ -173,21 +193,26 @@ func TestStackCreateInvalidStackName(t *testing.T) {
 	sandbox, cleanup := cmdtest.TestSetupWithSandbox(t, true)
 	defer cleanup()
 
-	err := os.RemoveAll("testing_stack")
-	if err != nil {
-		t.Fatal(err)
-	}
+	testStackName := "testing_stack_invalid_name"
 
-	args := []string{"stack", "create", "testing_stack"}
-	_, err = cmdtest.RunAppsody(sandbox, args...)
+	args := []string{"stack", "create", testStackName}
+	_, err := cmdtest.RunAppsody(sandbox, args...)
 
 	if err == nil {
 		t.Fatal(err)
```

Comment:
A better message here would be nice as to why the test is failing. The `err` will be `nil` at this point.